internal/server/evaluation/evaluation.go

```diff
@@ -91,3 +112,67 @@ func (s *Server) Boolean(ctx context.Context, r *rpcevaluation.EvaluationRequest
 
 	return resp, nil
 }
+
+// Batch takes ina list of *evaluation.EvaluationRequest and returns their respective responses.
+func (s *Server) Batch(ctx context.Context, b *rpcevaluation.BatchEvaluationRequest) (*rpcevaluation.BatchEvaluationResponse, error) {
+	resp := &rpcevaluation.BatchEvaluationResponse{
+		Responses: []*rpcevaluation.EvaluationResponse{},
+	}
+
+	for _, req := range b.GetRequests() {
+		f, err := s.store.GetFlag(ctx, req.NamespaceKey, req.FlagKey)
```

Comment:
future optimization, maybe we cache the flag in memory (in a map or whatever) and check the map before looking in the db, in case multiple requests are for the same flag

Reply:
@markphelps Makes sense. That in memory map would probably have to get instantiated for each `Batch` request. We most likely would not want to share the map across requests, right?

Reply:
right, for each batch request